src/app/features/products/views/ProductsActions.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button } from '~/src/app/shared/components/Button';
+import { ButtonContentWrapper } from '~/src/app/shared/components/Button/views/ButtonContentWrapper';
+import { ProductsActionsRootTv } from '../ProductsTV';
+import { Dispatch, SetStateAction } from 'react';
+
+interface ProductsActionsProps {
+  togglePress: () => void;
+  setIsTable: Dispatch<SetStateAction<boolean>>;
+}
+
+export function ProductsActions({ setIsTable, togglePress }: ProductsActionsProps) {
+  return (
+    <div className={ProductsActionsRootTv()}>
+      <Button.root
+        onMouseDown={togglePress}
+        onMouseUp={togglePress}
+        onClick={() => setIsTable((prev: any) => !prev)}

```

Comment:
Por que não tipar com `boolean`?

```suggestion
        onClick={() => setIsTable((prev: boolean) => !prev)}

```

Reply:
ops! está correto... este any está ai por equívoco. Antes de fazer o push, esta função recebia outras props e, como estava testando uma função, passei any para o es-lint parar de reclamar e o teste funionar, mas acabei mudando a estratégia e me esqueci deste any. Excelente observação